docs/fundamentals/executor/monitoring-executor.md

```diff
@@ -0,0 +1,168 @@
+(monitoring-executor)=
+# Monitor Executor with Custom Metrics
+
+Jina allows you to monitor every part of a Flow, including Executor, with the Grafana/Prometheus.
+This section documents the ability to add custom monitoring to the Executor.
+
+Custom metrics are useful when you want to monitor each subpart of your Executors. Jina allows you to leverage
+the full power of the [prometheus-client](https://github.com/prometheus/client_python) to define useful metrics 
```

Comment:
Should `prometheus` be capitalized here? Both P and C (first letters of each word) are capitalized in repo title